OracleAnalytics/README.md

```diff
@@ -0,0 +1,279 @@
+as starter templates to customize or tweak,
+or to create scripts and container files from scratch.
+
+For pre-built images containing Oracle software, check the [Oracle Container Registry](https://container-registry.oracle.com).
+
+## Database Prerequisites
+
+You must have a running Oracle Database, either in a container or on a host.
+You need these database connection details
+to create mid-tier schemas for the BI domain.
+The schemas are created automatically when the BI container starts.
+
+If you're using an Oracle multitenant container database (CDB)
+or pluggable database (PDB) database,
+you must use PDB when creating the schemas because CDB isn’t supported.
+
+You can create an Oracle Database container by using an
+[OracleDatabase](https://github.com/oracle/docker-images/tree/master/OracleDatabase)
+image.
+Follow these instructions to create a 12.1 or 12.2 Enterprise Edition database.
+
+## Oracle Analytics Server Container Image Creation
+
+To build a BI image, first build the Oracle Java image,
+and then build the Oracle Fusion Middleware Infrastructure image.
+
+### Building the Oracle Java (Server JRE 8) Image
+
+Download the Oracle Server JRE binary into folder `OracleJava/8` and build the image:
+
+```bash
+cd OracleJava/8
+docker build --tag oracle/serverjre:8 .
+```
+
+Refer to the README.md file under
+[OracleJava](https://github.com/oracle/docker-images/tree/master/OracleJava)
+for details on how to build Oracle Java image.
+
+### Building the Oracle Fusion Middleware Infrastructure Image
+
+Download the binary of Oracle Fusion Middleware Infrastructure into the folder `OracleFMWInfrastructure/dockerfiles/12.2.1.4`.
+
+If you need a proxy for the host to access yum.oracle.com during build,
+first set up the appropriate environment. For example:
+
+```bash
+export http_proxy=myproxy.example.com:80
+export https_proxy=myproxy.example.com:80
+export no_proxy="localhost,127.0.0.1,localaddress,.localdomain.com"
+```
+
+Build the image:
+
+```bash
+cd OracleFMWInfrastructure/dockerfiles
+./buildDockerImage.sh -v 12.2.1.4
+```
+
+Refer to the README.md file under
+[OracleFMWInfrastructure](https://github.com/oracle/docker-images/tree/master/OracleFMWInfrastructure)
+for details on how to build the Oracle Fusion Middleware Infrastructure image.
+
+### Building the Oracle Analytics Server Image
+
+Download the binaries for
+[Oracle Analytics Server 2022 (6.4)](https://www-sites.oracle.com/solutions/business-analytics/analytics-server/analytics-server.html)
+for Linux x86-64-bit into the folder `OracleAnalytics/dockerfiles/6.4`.
+
+If you need a proxy for the host to access yum.oracle.com during build,
+first set up the appropriate environment. For example:
+
+```bash
+export http_proxy=myproxy.example.com:80
+export https_proxy=myproxy.example.com:80
+export no_proxy="localhost,127.0.0.1,localaddress,.localdomain.com"
+```
+
+Build the image:
+
+```bash
+cd OracleAnalytics/dockerfiles
+./buildDockerImage.sh -v 6.4
+```
```

Comment:
Please also provide a `docker build` sample command for users who don't want to use this script.

Reply:
done.